erpnext/patches/v13_0/add_default_statement_of_account_email_template.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import unicode_literals
+import os
+
+import frappe
+from frappe import _
+
+
+def execute():
+	frappe.reload_doc("email", "doctype", "email_template")
+	frappe.reload_doc("accounts", "doctype", "accounts_settings")
+
+	if not frappe.db.exists("Email Template", _("Statement of Account")):
```

Comment:
```suggestion
	if not frappe.db.exists("Email Template", "Statement of Account"):
```

Reply:
@hrwX this added for translation of subject for email template